app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,79 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if session[:user_id]
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if session[:user_id]
+      erb :'tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    if params[:content] == ""
+      redirect to "/tweets/new"
+    else
+      user = User.find_by_id(session[:user_id])
+      @tweet = Tweet.create(:content => params[:content], :user_id => user.id)
+      redirect to "/tweets/#{@tweet.id}"
+    end
+  end
+
+  get '/tweets/:id' do
+    if session[:user_id]
```

Comment:
You can use your `logged_in?` helper method here, which is preferable to referring to the session object directly.